sorting-algorithms/insertion-sort.md

```diff
@@ -72,17 +83,35 @@ The insertion sort algorithm requires (in the worst case):
 - ... and so on
 - _n-1_ comparisons to insert the last element
 
-Overall, this is _1 + 2 + 3 + ... + (n-1)_, the same expression we saw for bubble sort and selection sort.
+Overall, this is _1 + 2 + 3 + ... + (n-1)_.
+
+There is a mathematical identity which states this summation is the same as _n(n-1)/2_. When multiplied out this results in _1⁄2n<sup>2</sup>-1⁄2n_, for _O(n<sup>2</sup>)_.
+
+Notice that _n(n-1)/2_ does _not_ equal _n<sup>2</sup>_. Recall that when performing big O analysis, we are interested in the rate of growth in the algorithm rather than the exact number of operations. The growth rate will be dominated by the largest term in the equation.
 
-Therefore insertion sort has a complexity of _O(n<sup>2</sup>)_.
+After multiplying out the expression, the largest term is _1⁄2n<sup>2</sup>_. After dropping the constant, this allows us to say that insertion sort is _O(n<sup>2</sup>)_.
 
-We might also notice that insertion sort, like bubble sort, has a best case time complexity of _O(n)_ if the list is already sorted. As we expand the sorted sub-part of the array, each new item is already in its correct spot, which we can determine with a single check per item. Since there are _n_ items, this gives a best case of _O(n)_.
+### !callout-info
+
+## Mathematical Proofs Out of Scope
+
+In general, full mathematical proofs for the algorithms we discuss are out of scope for the curriculum. We will often apply analyses that are more intuitive than rigorous. However, there are great resources online that explain the math in-depth! Follow your curiosity!
+
+### !end-callout
+
+Therefore insertion sort has a time complexity of _O(n<sup>2</sup>)_.
```

Comment:
Can we move this line above the callout? It feels like this got cut off from the preceding argument.